Tests/UITests/TestApp/HealthKitTestsView.swift

```diff
@@ -41,4 +44,42 @@ struct HealthKitTestsView: View {
             }
         }
     }
+    
+    func injectStepCountData() async {
+        // Generate sample data
+        guard let quantityType = HKQuantityType.quantityType(forIdentifier: .stepCount) else {
+            print("Step count quantity type not available.")
+            return
+        }
+        let HKStore = HKHealthStore()
+        let startDate = Date()
```

Comment:
If I understand this correctly, it's injected 500 step count samples that start now and end 5 minutes from now. 

Why does this get triggered in the bulk upload if it is fetching historical data from 6 months ago up until the time the app was first launched?